### PR TITLE
Time docs

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

File: 03.Devices/01.System-requirements/docs.md [hunk 1]
```diff
@@ -46,4 +46,22 @@ Please see [Partition layout](../partition-layout/).
...
 Certificate verification requires the device clock to be running correctly at all times.
 Make sure to either have a reliable clock or use network time synchronization.
-Please see [certificate troubleshooting](../../troubleshooting/mender-client#certificate-expired-or-not-yet-valid) for more information about the symptoms of this issue.
+Please see [certificate troubleshooting](../../troubleshooting/mender-client#certificate-expired-or-not-yet-valid) for more information about the symptoms of this issue. Note that the default setup of systemd will use network time
+synchronization to maintain the clock in a running system. This may
+take a few minutes to stabilize on system boot so it is possible
+to have a few connection rejections from the server until this process
+is complete and the time is correct.
+
+If your target does not have an active internet connection, then systemd
```
Comment (estenberg) [inline 03.Devices/01.System-requirements/docs.md]: "target" -> "device"

File: 03.Devices/01.System-requirements/docs.md [hunk 2]
```diff
@@ -46,4 +46,22 @@ Please see [Partition layout](../partition-layout/).
 ##Correct clock
 Certificate verification requires the device clock to be running correctly at all times.
 Make sure to either have a reliable clock or use network time synchronization.
-Please see [certificate troubleshooting](../../troubleshooting/mender-client#certificate-expired-or-not-yet-valid) for more information about the symptoms of this issue.
+Please see [certificate troubleshooting](../../troubleshooting/mender-client#certificate-expired-or-not-yet-valid) for more information about the symptoms of this issue. Note that the default setup of systemd will use network time
```
Comment (estenberg) [inline 03.Devices/01.System-requirements/docs.md]: Would have this sentence "Please see..." separately at the end here.

**Comment by estenberg** [comment]
Would also link to system-requirements#correct-clock from troubleshooting/mender-client#certificate-expired-or-not-yet-valid.

**Comment by drewmoseley** [comment]
@estenberg I pushed updates addressing your comments.  Please take a look.

**Pull request merged**